lib/t/push_notifications/apns.ex

```diff
@@ -121,8 +121,10 @@ defmodule T.PushNotifications.APNS do
   end
 
   def build_alert_payload("timeslot_accepted" = type, data) do
+    %{"name" => name} = data
+
     alert = %{
-      "title" => dgettext("apns", "Твое приглашение на дэйт принято"),
+      "title" => dgettext("apns", "Твое приглашение на дэйт c %{name} принято", name: name),
```

Comment:
doesn't seem to work:
![image](https://user-images.githubusercontent.com/16062322/141888237-1d307b82-9cbb-46e9-9b06-4e6a8fc98cbd.png)

Reply:
<img width="570" alt="image" src="https://user-images.githubusercontent.com/90351917/142064175-08f299b4-4050-4a67-90c9-12c23b0c4753.png">

Reply:
it works now, thanks :)